docs/manifest.json

```diff
@@ -0,0 +1,260 @@
+{
+  "lang": "node",
+  "friendlyLang": "Node.js",
+  "markdown": "javascript",
+  "versions": [
```

Comment:
@stephenplusplus any thoughts on how we should go about programmatically updating this?